velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -55,20 +55,33 @@ ::substrait::Plan& VeloxToSubstraitPlanConvertor::toSubstrait(
   exprConvertor_ =
       std::make_shared<VeloxToSubstraitExprConvertor>(functionMap_);
 
-  // TODO add root_rel
   ::substrait::Plan* substraitPlan =
       google::protobuf::Arena::CreateMessage<::substrait::Plan>(&arena);
 
   // Add Extension Functions.
   substraitPlan->MergeFrom(addExtensionFunc(arena));
 
   // Do conversion.
-  ::substrait::Rel* rel = substraitPlan->add_relations()->mutable_rel();
-  toSubstrait(arena, plan, rel);
+  ::substrait::RelRoot* rootRel =
+      substraitPlan->add_relations()->mutable_root();
+  toSubstrait(arena, plan, rootRel);
 
   return *substraitPlan;
 }
 
+void VeloxToSubstraitPlanConvertor::toSubstrait(
```

Comment:
This method is very short and used only once. Let's fold it into the caller.

Reply:
done.